circuit_passes/src/passes/unknown_index_sanitization.rs

```diff
@@ -154,6 +154,9 @@ impl UnknownIndexSanitizationPass {
          *      (base_offset + (mul_offset * UNKNOWN))
          * So, if we set the unknown value to 0, we will compute the base offset,
          * which will let us look up the range of the underlying array.
+         *
+         * This specifically tackles 1-D arrays (what we currently deal with), but could be
+         * extended to arbitrary dimensions if needed.
```

Comment:
No, what I mean is that `(base_offset + (mul_offset * UNKNOWN))` is for a 1D array and if it where a 2-D array is a different expression that works very similarly.

```suggestion
         *
         * The expression above is for 1-D arrays, multidimensional arrays follow 
         * a similar pattern that is also handled here.
```